migrations/postgres/9_add_index_replication_support.up.sql

```diff
@@ -0,0 +1,199 @@
+-- Migration: Add full index replication support (triggers, log_schema, get_schema)
```

Comment:
You need to run `make gen-migrations` so that the migrations are added to the bindata that is later applied within the init/destroy migrator. 
https://github.com/xataio/pgstream/blob/main/migrations/postgres/migrations.go